.golangci.yml

```diff
@@ -113,3 +113,6 @@ issues:
     - path: _test\.go
       linters:
         - scopelint
+    - path: perf_scenarios\.go
```

Comment:
Why is this needed? `testbed` directory is already skipped entirely in this config. Which raises the question: why is it skipped? Perhaps enable it instead of having linting of the repo done by 2 separate makefiles?

Reply:
From the PR description:
> Since testbed has its own go module and golangci uses go/analysis,
testbed folder will be excluded from linting

That's why we can safely remove testbed ignore from the config, too.

This file is ignored because it follows testing convetions (subtest) and has false positives for scopelint (https://github.com/kyoh86/scopelint/issues/4).

Regarding two makefile, second makefile was already there, instead of hardcoding path in the first makefile, use the second to run the command. Let me know if I need to drop lint target from the second makefile and only use the first.

Reply:
I am probably missing something.

If entire `testbed` folder is skipped why is there a need to have a setting for `perf_scenarios.go` which is in that folder. Isn't `perf_scenarios.go` ignored anywhere because `testbed` is skipped (line 20 of `.golangci.yml`)?

Reply:
Previously, there was no go module in testbed so it was skipped via configuration. Right now, it will skipped by default because it's a different module. 

Additionally, we're running golangci second time in testbed folder so this configuration doesn't apply and actually it could safely be removed.